migrations/20191014160144_add_project_amended_date.js

```diff
@@ -0,0 +1,8 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.table('projects', table => table.dateTime('amended_date'));
+};
+
+exports.down = function(knex, Promise) {
+  return knex.schema.table('projects', table => table.dropColumn('amended_date'));
+};
```

Comment:
This already exists in `master` so I'm not sure why it isn't hidden in this diff?

Reply:
I'll rebase